org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -8,14 +8,13 @@
 var request = require('request-promise');
 var OpenT2T = require('opent2t').OpenT2T;
 var Crypto = require('crypto');
-var accessTokenInfo = require('./common').accessTokenInfo;
 
 /**
 * This translator class implements the "Hub" interface.
 */
 class Translator {
-    constructor(accessToken) {
-        this._accessToken = accessToken;
+    constructor(authTokens) {
+        this._authTokens = authTokens;
```

Comment:
Yes, though I was not able to use the CLI to control 2 of each type of device for all providers (Insteon) so I do want to do a little more verification there.

---
In reply to: [101103263](https://github.com/openT2T/translators/pull/160#discussion_r101103263) [](ancestors = 101103263,101102384,101099112)